### ISSUE TITLE
Clicking a phone number should copy its value

### ISSUE THREAD

**Issue by Bonapara** [twentyhq/twenty#8905]
# Current behavior

Clicking on a phone number navigates to the phone app

https://github.com/user-attachments/assets/2684c5d9-1595-4277-afe9-3957c4b8d27c

# Desired Behavior

The default behavior should be to copy the phone number to the clipboard and display a notification

![Image](https://github.com/user-attachments/assets/2161fddd-91fa-4d6b-9464-4754d5f8cf86)

**Comment by harshrajeevsingh** [comment]
@Bonapara This should be the default behaviour only for the record table or everywhere across the app?

**Comment by shivankurchavan** [comment]
i want to work on this issue

**Comment by Bonapara** [comment]
Maybe you can do some peer coding @shivankurchavan and @Lucifer4255?

**Comment by Lucifer4255** [comment]
@shivankurchavan up for peer coding?

**Comment by harshrajeevsingh** [comment]
@Lucifer4255 I guess there is already a component called snackbar.

**Comment by Bonapara** [comment]
Right! https://storybook.twenty.com/?path=/story/ui-feedback-snackbarmanager-snackbar--default

**Comment by Lucifer4255** [comment]
@Bonapara Since no one is working on this issue i guess can you assign it to me im merging a PR for closing this issue you can check there

**Comment by shivankurchavan** [comment]
> [@shivankurchavan](https://github.com/shivankurchavan) up for peer coding?

Good job fixing this issue, kudos.
Can you help me in setting up the project ? Im facing some issues
